child_compassion/wizards/global_child_search.py

```diff
@@ -113,6 +116,75 @@ def rich_mix(self):
         return True
 
     @api.multi
+    def make_a_hold(self):
+        """ Create hold and send to Connect """
+        holds_ids = []
+        messages_ids = []
+        for child in self.global_child_ids:
+            # Save children form global children to compassion children
+            child_vals = {
+                'global_id': child.global_id,
+                'local_id': child.local_id,
+                'project_id': '' if child.project_id is None
+                else child.project_id.id,
+                'field_office_id': child.field_office_id,
+                'name': child.name,
+                'firstname': child.firstname,
+                'lastname': child.lastname,
+                'preferred_name': child.preferred_name,
+                'gender': child.gender,
+                'birthdate': child.birthdate,
+                'age': child.age,
+                'is_orphan': child.is_orphan,
+                'beneficiary_state ': child.beneficiary_state,
+                'sponsorship_status': child.sponsorship_status,
+                'unsponsored_since': child.unsponsored_since,
+            }
+            child_comp = self.env['compassion.child'].create(child_vals)
+
+            # Create Holds for children to reserve
+            hold_vals = {
+                'name': "",
+                'child_id': child_comp.id,
+                'type': 'Consignment Hold',
+                'expiration_date': self.hold_expiration_date,
+                'primary_owner': 'Rose-Marie Reber',
+                'secondary_owner': 'Carole Rochat',
+                'no_money_yield_rate': '1.1',
+                'yield_rate': '1.1',
+                'channel': '',
+                'source_code': '',
+            }
+            hold = self.env['compassion.hold'].create(hold_vals)
+            holds_ids.append(hold.id)
```

Comment:
You can keep a recordset directly instead of saving the ids and then call the `browse` method. It is done this way:
```
holds = self.env['compassion.hold']
holds += holds.create({...})
```